common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -2832,3 +2832,60 @@ func (d *cassandraPersistence) GetReplicationTasksFromDLQ(
 
 	return d.populateGetReplicationTasksResponse(query)
 }
+
+func (d *cassandraPersistence) DeleteReplicationTaskFromDLQ(
+	request *p.DeleteReplicationTaskFromDLQRequest,
+) error {
+
+	query := d.session.Query(templateCompleteTransferTaskQuery,
```

Comment:
for clarity, can you either (1) create an alias for templateCompleteTransferTaskQuery or (2) rename templateCompleteTransferTaskQuery to templateCompleteTaskQuery ?

Reply:
Updated to templateCompleteReplicationTaskQuery